packages/screens/TeritoriNameService/TNSHomeScreen.tsx

```diff
@@ -141,7 +142,11 @@ export const TNSHomeScreen: ScreenFC<"TNSHome"> = ({ route }) => {
   return (
     <ScreenContainer
       noMargin={width <= 1600}
-      headerChildren={<BrandText>Name Service</BrandText>}
+      headerChildren={
+        <BrandText style={fontSemibold20}>Name Service</BrandText>
+      }
+      responsive
+      onBackPress={() => navigation.navigate("TNSHome")}
```

Comment:
We already are on TNSHome.
Maybe should we navigate to "Home" on root feature's screens ?